libr/debug/debug.c

```diff
@@ -1511,24 +1510,13 @@ R_API ut64 r_debug_get_baddr(RDebug *dbg, const char *file) {
 		// Tell gdb that we want baddr, not full mem map
 		dbg->iob.system(dbg->iob.io, "baddr");
 	}
-#if __WINDOWS__
```

Comment:
@xarkes said he is refactoring the windbg, this code is a very good example of things that should live in a plugin, but doesn't.